code/hud/hudtarget.cpp

```diff
@@ -5533,6 +5533,16 @@ void HudGaugeWeaponEnergy::initBitmaps(char *fname)
 	Energy_bar.first_frame = bm_load_animation(fname, &Energy_bar.num_frames);
 	if ( Energy_bar.first_frame < 0 ) {
 		Warning(LOCATION,"Cannot load hud ani: %s\n", fname);
+	} else {
```

Comment:
could you confirm that this is the only non-whitespace change in the PR?  This chunk looks fine to me, but I just want to be sure I didn't miss anything.